spec/datetimeformat.html

```diff
@@ -550,6 +558,7 @@ <h1>Intl.DateTimeFormat.prototype.resolvedOptions ( )</h1>
             1. Else, let _v_ be *undefined*.
           1. Else,
             1. Let _v_ be the value of _dtf_'s internal slot whose name is the Internal Slot value of the current row.
+            1. If _p_ is *"fractionalSecondDigits"*, set _v_ to 𝔽(_v_).
```

Comment:
Using Numbers instead of mathematical values for the internal slots means the special treatment will just move to other places which then have to apply ℝ, for example see [FormatNumericToString](https://tc39.es/ecma402/#sec-formatnumberstring). 

I tried both alternatives, but eventually settled on using mathematical values for the internal slots, because that matches what ECMA-262 is doing and it reduces the ℝ&rarr;𝔽 conversions to built-in functions, whereas all abstract operations can directly perform any operations using the slot values without having to perform additional ℝ conversions.

Reply:
In that case, I would like to see Conversion columns (as in [Temporal](https://tc39.es/proposal-temporal/#table-temporal-field-requirements)) to avoid mentioning properties by name in algorithm steps.

Reply:
Fixed.